spk/pyload/src/dsm-control.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# Package
+PACKAGE="pyload"
+DNAME="pyLoad"
+
+PIDFILE="/var/run/pyload.pid"
+PYLOADCORE="/usr/local/python/bin/python /usr/local/pyload/share/pyload/pyLoadCore.py --pidfile=${PIDFILE}"
```

Comment:
Change this to `PYLOADCORE = "${INSTALL_DIR}/share/pyload/pyLoadCore.py`, and call something like this in the start function:
`${PYTHON} ${PYLOAD} --pidfile ${PIDFILE}`

Along with that, you should use a Python virtual environment instead of the system Python (to be created during install). Other Python-based packages could serve as an example.